app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -136,7 +128,7 @@ def prepare_for_evss!
       Rails.logger.error e.backtrace.join('\n')
     end
 
-    prepare_for_ep_merge! if eligible_for_ep_merge? && is_claim_fully_classified
```

Comment:
I wrote `#eligible_for_ep_merge?` specifically to keep all eligibility rollout logic in one place, so that after rollout is finished, all that's needed is to collapse this to `true` and remove the logic and tests.